src/profile/components/SwitchOrganization/SwitchOrganization.tsx

```diff
@@ -0,0 +1,37 @@
+import { Flex, FormSelect, Copy, Spacer } from "@/shared/components"
+import { OrganizationType } from "@/profile"
+
+interface ISwitchOrganizationProps {
+  setUserOrganization: Function
+  userOrganization: OrganizationType | string
+}
+
+const organizationTypeList: OrganizationType[] = Object.values(OrganizationType)
```

Comment:
change to `const organizationTypeList: string[] = ["1", "2", "3"]`

Reply:
Done